docker-compose.yml

```diff
@@ -1,4 +1,3 @@
-version: '3.8'
```

Comment:
Is it a good practice to remove the version? I mean, how can `docker compose` know what is the version of the file? Some configuration changes over the versions, so I questioning this change.

Reply:
Oh, I so the justification in the commit.

Reply:
It's nice they changed that behavior.

Reply:
I agree, I was a bit worried but it seems it is now the way to go; they keep it to be backward compatible but they recommend to remove it (docker compose will always assume you are on the latest version)
See: https://docs.docker.com/compose/compose-file/04-version-and-name/#version-top-level-element-obsolete

> Compose doesn't use version to select an exact schema to validate the Compose file, but prefers the most recent schema when it's implemented.

I'm not sure it is a good move from them, but this is how it works now 😅